src/transformers/modeling_utils.py

```diff
@@ -2107,7 +2122,6 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
                 )
                 raise
         elif from_pt:
-
```

Comment:
Loos unrelated to SMP, so let's leave it as before (feel free to make another PR focused on style changes!)

Reply:
Yeah sounds good. These line removals/additions were essentially done on accident while I was going through the code!

Reply:
Oh actually, I think many of these line changes were done by running `black --preview` on the file (as indicated in the contributor guidelines). Should I just not run it then? Thanks!

Reply:
If you make sure you have the version of black pinned by Transformers (22.3), you shouldn't have those.

Reply:
Oh yeah my bad, I was using black 22.6!